meson.options

```diff
@@ -21,3 +21,8 @@ option(
     value: 'disabled',
     description : 'Enable OperationalStatus monitor'
 )
+option(
```

Comment:
you removed monitor-sai-status from these two commits "meson: set use-json as default" and "format meson_options file"

and added it in this commit "meson_options.txt: Support for reading options from meson.options"

Please revert the removal.

Reply:
ACK